src/util/s3select.js

```diff
@@ -60,6 +93,7 @@ class S3SelectStream extends Transform {
     */
     constructor(opts) {
         super(opts);
+        this.stats = { Stats: { BytsProcessed: 0, BytesScanned: 0, BytesReturned: 0 } };
```

Comment:
Instead of `BytsProcessed` `BytesProcessed`.
Would you please change this object to appear on multi-lines? (add new line after each property, it will be easier to read)